packages/ndla-licenses/src/licenses.ts

```diff
@@ -57,7 +57,7 @@ const byncnd: LicenseType = {
       'This license is the most restrictive of our six main licenses, only allowing others to download your works and share them with others as long as they credit you, but they can’t change them in any way or use them commercially.',
   },
   rights: [CC, BY, NC, ND],
-  abbreviation: `${CC} ${BY}-${NC}-${ND}`.toUpperCase(),
+  abbreviation: `${CC} ${BY}-${NC}-${ND} 4.0`.toUpperCase(),
```

Comment:
Dette VIL knekke ting. Funksjonen getLicenseByAbbreviation brukes for å mappe lisenskode fra backend til en lisens og om du endrer her vil det slutte å fungere.

Reply:
Har pushet nye endringer. Reverserte endringene i denne filen og gjør transformering lokalt i getCopyString-funksjonene så de blir på riktig format.